doc/foreign-code.rst

```diff
@@ -160,8 +160,8 @@ Your example ``function_description.ml`` file is:
 
   open Ctypes
 
-  (* This Types_generated module is an instantiation of the Types
-     functor defined in the type_description.ml file. It's generated by
+  (* This Types_generated module is an instantiation of the ``Types``
+     functor defined in the ``type_description.ml`` file. It's generated by
```

Comment:
In the indented section, we are actually writing OCaml source. So ReStructured Text's conventions no longer apply. Therefore the quoting does nothing here and should be removed.